fvtest/compilerunittest/p/BinaryEncoder.cpp

```diff
@@ -2669,6 +2669,13 @@ INSTANTIATE_TEST_CASE_P(VSXVectorFixed, PPCTrg1Src3EncodingTest, ::testing::Valu
     std::make_tuple(TR::InstOpCode::xxsel, TR::RealRegister::vsr0,  TR::RealRegister::vsr0,  TR::RealRegister::vsr0,  TR::RealRegister::vsr63, TRTest::BinaryInstruction("f00007f8"))
 ));
 
+INSTANTIATE_TEST_CAST_P(VSXVectorFixed, PPCTrg1ImmEncodingTest, ::testing::Values(
+    std::make_tuple(TR::InstOpCode::xxspltib, TR::RealRegister::vsr31, 0x00000000u, TRTest::BinaryInstruction(0xf3e002d0u)),
+    std::make_tuple(TR::InstOpCode::xxspltib, TR::RealRegister::vsr0,  0x0000000fu, TRTest::BinaryInstruction(0xf00f02d0u)),
+    std::make_tuple(TR::InstOpCode::xxspltib, TR::RealRegister::vsr15, 0xfffffff0u, TRTest::BinaryInstruction(0xf1f002d0u)),
+    std::make_tuple(TR::InstOpCode::xxspltib, TR::RealRegister::vsr16, 0xffffffffu, TRTest::BinaryInstruction(0xf21f02d0u))
```

Comment:
This still seems to be using the (since-removed) `uint32_t` constructors for `BinaryInstruction` as @BradleyWood pointed out. The new version of the binary encoder test framework (as of #6216) expects a hex string IIRC.